bucket/xmind-zen.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "10.0.1",
+    "description": "XMind is the most professional and popular mind mapping tool.",
+    "homepage": "https://www.xmind.net/zen/",
+    "license": {
+        "identifier": "Shareware",
+        "url": "https://www.xmind.net/terms/"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://dl2.xmind.cn/XMind-ZEN-for-Windows-64bit-10.0.1-202001022330.exe#/dl.7z",
+            "hash": "298d2e0eddc3fbb897ac134d6a2d70274a821d490dc7e4b84cd560a63de429ce",
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                    "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Force -Recurse"
+                ]
+            }
+        }
+    },
```

Recalculate hashes and try with this

```suggestion
        "64bit": {
            "url": "https://dl2.xmind.cn/XMind-ZEN-for-Windows-64bit-10.0.2-202002120025.exe#/dl.7z",
            "hash": "asdfasdfasd",
            "installer": {
                "script": "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\""
            }
        },
        "32bit": {
            "url": "https://dl2.xmind.cn/XMind-ZEN-for-Windows-32bit-10.0.2-202002120038.exe#/dl.7z",
            "hash": "faskdjfsdla",
            "installer": {
                "script": "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-32.7z\" \"$dir\""
            }
        }
    },
    "post_install": "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse",
```